addons/website_forum/tests/test_forum.py

```diff
@@ -19,22 +19,21 @@ def test_ask(self):
                 'forum_id': self.forum.id,
             })
 
-        # Portal user asks a question with tags: not allowed, unsufficient karma
+        # Portal user asks a question without tags: not allowed, unsufficient karma
```

Comment:
when portal user asks a question with insufficient karma, tags doesn't matter whether tag are there in question or not because it has insufficient karma for asking question. 
Previously 'unique tag creation' point is there in spec. so better to remove tag_ids in post creation.